lucille-core/src/main/java/com/kmwllc/lucille/connector/jdbc/DatabaseConnector.java

```diff
@@ -197,32 +208,32 @@ private void iterateOtherSQL(ResultSet rs2, String[] columns2, Document doc, Int
     // Test if we need to advance or if we should read the current row ...
     if (rs2.isBeforeFirst()) {
       // we need to at least advance to the first row.
-      rs2.next();    
+      rs2.next();
     }
-    // Test if this resultset is alread exhausted.
+    // Test if this result set is already exhausted.
     if (rs2.isAfterLast()) {
-      // um.. why is this getting called?  if it is?
+      // um… why is this getting called?  if it is?

```

Comment:
remove the comment; try to avoid introducing special chars like U+2026 unless there's a need